neutronics/mcnp_inputs.py

```diff
@@ -0,0 +1,157 @@
+import math
+from string import Template
+import material_data as md
+
+def merge_comps(compA, compB):
```

Comment:
How do you account for there being different volume fractions during homogenization?

Reply:
This function is not supposed to weight masses with volumes. It takes the weighted compositions and merges them. Maybe I am not understanding your question?

Reply:
@gonuke I didn't see your main comment above earlier. I understand what you mean about the convoluted steps. This is always how I thought of the homogenization process in my head. I see how it makes sense to change this up I'll rework this.

Reply:
I made a minor change, I separated the normalization step from the isotopic mass calculation. The steps are now this:
* Using volfrac and density, get isotopic mass vector for all 4 components
* Merge all 4 components
* Normalize the final composition with the mass sum of all isotopes.